api/src/main/java/jakarta/data/metamodel/NavigableAttribute.java

```diff
@@ -21,8 +21,28 @@
 
 import java.util.Objects;
 
+/**
+ * <p>Represents an entity attribute that is an embeddable or relation.
```

Comment:
```suggestion
 * <p>Represents an entity attribute that is an embeddable or association to another entity.
```

"Association" is the term used in the spec.